pref/pref.go

```diff
@@ -0,0 +1,73 @@
+package pref
+
+import (
+	"gioui.org/x/pref/internal/battery"
+	"gioui.org/x/pref/internal/locale"
+	"gioui.org/x/pref/internal/theme"
+	"golang.org/x/text/language"
+)
+
+var (
+	// DefaultLanguage defines the default language, used by default when impossible to get the preferred language.
+	DefaultLanguage = language.English
+
+	// DefaultTimeFormat defines the default date format.
+	// DefaultTimeFormat = "2006-01-02 15:04:05"
+)
+
+// LocaleInfo have information about the language of the user.
+type LocaleInfo struct {
+	// Language is the preferred language of the end-user or the language of the operational system.
+	// It will fallback to DefaultLanguage when not possible to determine the language.
+	Language language.Tag
+
+	// TimeFormat is the date format of the current language.
+	// TimeFormat string
+}
+
+// Locale returns information about the language, accessible via LocaleInfo.
+func Locale() LocaleInfo {
+	return LocaleInfo{
+		Language: locale.Language(DefaultLanguage),
+	}
+}
+
+// BatteryInfo have information about the battery of the device and related user preferences.
+type BatteryInfo struct {
+	// Level represents the battery level as percent level, between 0~100.
+	// If the device doesn't run on battery it will return 100.
```

Comment:
Some APIs (Javascript) doesn't expose that information. I don't know about Android, since most Android devices uses batteries, so maybe it could be hardcode to `true`?! 🤔

The Windows API have that information, so it could be added.